### ISSUE TITLE
Asking for using of zenchrome sync principle.

### ISSUE THREAD

**Issue by Aleod-m** [george-b/zenchrome#2]
Hi,

I'm currently creating my own theme based on the monokai theme and i would like to use your method for syncing all the colors. As you didn't license your code and don't have a way to contact you directly i'm asking here if i can use your project as a "template" for mine here. Of course i would mention your implementation in the readme.

Kind regards.

**Comment by g0xA52A2A** [comment]
I've added a permissive license to clarify things. Initially I didn't bother including one as I don't think simple editor plugins are that novel.

As for custom themes in 2b93dae I removed the option for passing in a dictionary as a static data structure precludes the possibility changing when `'background'` is changed. I've just pushed a branch called "custom" in which 550be04 adds support for the user defining a function `Colors()` to set `g:Colorscheme` which would resolve the `'background'` issue. Though I'm unsure of how clean this really is, feedback is welcomed.
